ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.service.ts

```diff
@@ -1,119 +1,140 @@
-import {Trip} from "./trips";
-import {TripStatus} from "../../enums/TripStatus";
-import {UserState} from "../../enums/UserState";
+import {Trip} from './trips';
+import {TripStatus} from '../../enums/TripStatus';
+import {UserState} from '../../enums/UserState';
+import {FormControl} from '@angular/forms';
 
 export class TripsModalService {
 	constructor() {}
 
+	roles = [{value: 0, viewValue: 'Passenger'}, {value: 1, viewValue: 'Driver'}, {value: 2, viewValue: 'All'}];
+	statuses = new FormControl();
```

Comment:
move to the component statuses = new FormControl(); ratings = new FormControl();

Reply:
okay